code/datums/components/food/edible.dm

```diff
@@ -28,6 +28,8 @@ Behavior that's still missing from this component that original food items had t
 	var/junkiness = 0
 	///Message to send when eating
 	var/list/eatverbs
+	///Callback to be ran to see if the food can be eaten. Specifically used for carbon handling.
+	var/datum/callback/check_edible
```

Comment:
This is currently unused, any plan for it?

Reply:
Yup, I do have some ideas with some specific antag/species only edibles, but I can also just remove it and add it with the PR implementing said ideas. Just felt weird that it allows for different custom effects but not for actually custom checking if you can eat it and whatnot

Reply:
If it's something for the future, then don't include this change in this PR. It's pretty small and doesn't affect this feature in any way..

Reply:
Alright, fixed up